docs/INSTALL.md

```diff
@@ -0,0 +1,179 @@
+7. You're all done. Python is installed and ready to use.
+
+### Option 2: Install with Homebrew
+[Homebrew](https://brew.sh/) is a MacOS Linux-like package manager. Walk through the below steps to install Homebrew and an updated Python interpreter along with it.
+1. Open your *Terminal* application and run: xcode-select --install. This will open a window. Click *'Get Xcode'* and install it from the app store.
+2. Install Homebrew. Run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+   - You can also find this command on the [Homebrew website](https://brew.sh/)
+3. Install latest Python3 with brew install python
+4. Once Python is installed, you should be able to open your *Terminal* application, type python3, hit ENTER, and see a Python 3.X.X prompt opened. Type quit() to exit it. You should also be able to run the command pip3 and see its options. If both of these work, then you are ready to go
+   - Here are some additional resources on [Installing Python 3 on Mac OS X](https://docs.python-guide.org/starting/install3/osx/)
+
+<div style="display: flex; align-items: center;">
+  <img src="https://upload.wikimedia.org/wikipedia/commons/a/af/Tux.png" alt="Tux Linux Mascot" width="50">
+  <h2 style="margin-left: 10px;">Linux</h2>
+</div>
+
+- *Raspberry Pi OS* may need Python and PIP
+  - Install them: sudo apt install -y python3-pip
+- *Debian (Ubuntu)* distributions may need Python and PIP
+  - Update the list of available APT repos with sudo apt update
+  - Install Python and PIP: sudo apt install -y python3-pip
+- *RHEL (CentOS)* distributions usually need PIP
+  - Install the EPEL package: sudo yum install -y epel-release
+  - Install PIP: sudo yum install -y python3-pip    
+- *Fedora* distributions may need Python and PIP
+  - Install Python and PIP: `sudo dnf install -y python3 python3-pip`
+- *Arch Linux* may need Python and PIP
+  - Update the package database: `sudo pacman -Sy`
+  - Install Python and PIP: `sudo pacman -S python python-pip`
+
+
+<div style="display: flex; align-items: center;">
+  <img src="https://raw.githubusercontent.com/rootCircle/codemark-cli/main/logo.png" alt="Codemark CLI Logo" width="100">
+  <h1 style="margin-left: 10px;">Install Codemark CLI</h1>
+</div>
+
+
+## Installation instructions
+
+To install and use CodeMark CLI, follow these steps:
+
+1. Clone the CodeMark CLI repository by running the following command:
+ 
+    ```bash
+    git clone https://github.com/rootCircle/codemark-cli.git
+    ```
+
+2. Install CodeMark CLI by running the following commands:
+    ```bash
+     cd codemark-cli
+     pip3 install poetry # use pip if pip3 is not available
+     poetry install # python3 -m poetry install
+     pip3 install --editable .
+    ```
+     Note: If pip3 is not installed on your system, use pip instead. If you encounter permission issues in `pip3 install --editable .`, add the --user flag at the end of the command. 
```

Comment:
Present all topics of Note and other, as sub-point of 2nd step.

Something like

2. Some heading

- Subtopic